tests/expectations/snapshots/snap_test_expectation_atomic_renderers.py

```diff
@@ -348,12 +330,10 @@
         "header": None,
         "params": {
             "column": {"schema": {"type": "string"}, "value": "my_column"},
-            "condition_parser": {"schema": {"type": "string"}, "value": None},
-            "row_condition": {"schema": {"type": "string"}, "value": None},
             "ties_okay": {"schema": {"type": "boolean"}, "value": True},
-            "v__0": {"schema": {"type": "string"}, "value": 1},
-            "v__1": {"schema": {"type": "string"}, "value": 2},
-            "v__2": {"schema": {"type": "string"}, "value": 3},
+            "v__0": {"schema": {"type": "number"}, "value": 1},
+            "v__1": {"schema": {"type": "number"}, "value": 2},
+            "v__2": {"schema": {"type": "number"}, "value": 3},
```

Comment:
Not blocking, just curious how this worked before with the type set to `string`?

Reply:
Cloud just rendered the number as a string, which is fine as long as they don't need to do any math with it.